lib/index.js

```diff
@@ -21,9 +21,16 @@ function start(config) {
 
     debuggerWrapper.init();
 
-    setInterval(() => {
-      debuggerWrapper.handlePeriodicTasks();
-      transmitter.handlePeriodicTasks(config);
+    const periodicInterval = setInterval(() => {
+      try {
+        debuggerWrapper.handlePeriodicTasks();
+        transmitter.handlePeriodicTasks(config);
+      } catch (error) {
+        try {
+          clearInterval(periodicInterval);
+          console.log('Error in Snyk runtime agent, please contact runtime@snyk.io', error);
+        } finally {}
```

Comment:
What does an empty `finally {}` block do?

Reply:
absolutely nothing (swallows exceptions).
this is a just-in-case precaution if `clearInterval` (or for whatever reason `console.log`) throws an exception.

Reply:
this is just like an empty `catch` clause, only `catch` forces you to accept the error argument so `finally` felt cleaner